modules/stanford_profile_styles/lib/scss/paragraph-types/wysiwyg.scss

```diff
@@ -59,136 +59,150 @@
     max-width: 78ch;
   }
 
-  // Lists styles
-  ul,
-  ol {
-    @include big-paragraph;
-  }
-
-  // Caption Styles.
-  .caption {
-    figcaption {
-      text-align: center;
-    }
-  }
-
-  // Paragraph styles.
+  // Default styles.
   p {
     @include big-paragraph;
     @include modular-spacing('padding-bottom', 1);
     @include margin(0 null null null);
-   }
 
-  // Buttons.
-  .su-button {
-    @include modular-spacing('margin-bottom', 0);
-  }
-
-  // Bigger buttons.
-  .su-button--big {
-    @include modular-spacing('margin-bottom', 0);
-  }
-
-  // Secondary buttons.
-  .su-button--secondary {
-    @include modular-spacing('margin-bottom', 0);
-  }
-
-  // Big bold beautiful callout text.
-  .su-callout-text {
-    @include margin(0 auto 1em 0);
-    @include modular-spacing('margin', -3 null 1 null);
-    display: block;
-    font-size: 28px;
-    font-style: normal;
-    font-weight: bold;
-    line-height: 33px;
-
-    // 60 character line length.
-    max-width: 45ch;
-    width: 80%;
+    // Caption Styles.
+    @at-root .caption,
+    &.caption {
+      figcaption {
+        text-align: center;
+      }
+    }
 
-    &:first-child {
-      // Remove margin-top and padding-top when it is the first item on the page
-      @include margin(0 null null null);
-      @include padding(0 null null null);
+    // Buttons.
+    @at-root .su-button,
```

Comment:
`@at-root` is not what we need here as it refers to `body`

This statement outputs the following selector:
```
body .su-button,
.su-wysiwyg-text .su-button {
...
}
```

We want to output:
```
.su-wysiwyg-text .su-button,
.su-wysiwyg-text p.su-button {
...
}
```